ui/package.json

```diff
@@ -5,7 +5,7 @@
   "dependencies": {
     "@fortawesome/fontawesome-free": "^5.7.2",
     "@kubernetes/client-node": "github:scality/kubernetes-client-javascript.git#browser-0.10.2-62-g61d8543",
-    "@scality/core-ui": "github:scality/core-ui.git#094b88f",
+    "@scality/core-ui": "github:scality/core-ui.git#feature/improve-build-process",
```

Comment:
I don't really like it when we rely on branches in remote repositories.

Reason is simple: if the branch is removed from the remote repository, the build here will break. Similarly, when doing a `git bisect` for whatever reason, a rebuild of an old revision may not be possible.

Unless `package-lock.json` is the source of truth and the branch information is not actually used after the lock-file is generated?

Reply:
This branch is for test. when it is merged, we will update it with the development/1.0 latest commit hash.